app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -193,6 +203,9 @@ class HomeMenu(
             syncedTabsItem,
             bookmarksItem,
             historyItem,
+            BrowserMenuDivider(),
+            desktopItem,
```

Comment:
I am curious what designs we are using since the home menu ordering doesn't look anything like this anymore.

Reply:
I am just inserting this item here, I am somewhat unsure on the final designs and if there is any open issue about reordering menu items in the home screen,

Reply:
The rest of the reordering is in https://github.com/mozilla-mobile/fenix/issues/17770